src/thor/timedep_forward.cc

```diff
@@ -1,9 +1,9 @@
 #include "baldr/datetime.h"
+#include "baldr/graphconstants.h"
```

Comment:
everything here was stuff that astar was filling in for timedep_forward. instead of keeping astar around as pure virtual i just moved its guts to here.